### DISCUSSION TITLE
More examples link doesn't reference anything

### DISCUSSION THREAD

**Discussion by dreckguy** [QualiTorque/qualitorque.github.io#50] [Documentation requests] [answered]
In [The Bluprint Yaml Page](https://docs.qtorque.io/blueprint-designer-guide/blueprints) there is a sentence:
> "More examples for templating will be described HERE LINK"
But it doesn't link anything :
![broken-link](https://user-images.githubusercontent.com/15980885/175812709-b64a5a7b-d07a-405b-bcca-948e900693d8.png)

**Comment by Shnitselon** [comment, marked as answer]
Hi Ophir,

Thanks for the feedback.
Fixed.
